RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -889,22 +881,53 @@ boolean forbidFamChange()
 	{
 		return false;
 	}
+	
+	// if you reached this line, then it means that auto_100familiar is set to some specific familiar.
 	return true;
 }
 
-boolean forbidFamChange(familiar thisOne)
+boolean forbidFamChange()
```

Comment:
I'd also rename this to `canChangeFamiliar()` and invert your return values.

Reply:
This would require a seperate PR where I go through every single instance of its use in the code and either add or subtract a ! to flip it from its current use.
Entirely doable. is everyone else for it too?

Reply:
I agree `canChangeFamiliar()` is clearer, I support

Reply:
Alright. and thinking about it some more i am also in favor.
once this PR is resolved ill start a new PR specifically for the rename. (don't want rename PRs to languish too long. since they break too easily due to touching too many files and need constant refactoring)